homeassistant/config.py

```diff
@@ -630,6 +654,27 @@ def set_time_zone(time_zone_str: Optional[str]) -> None:
             ", ".join('{}: {}'.format(key, val) for key, val in discovered))
 
 
+async def async_store_ha_core_config(hass: HomeAssistant) -> None:
```

Comment:
No one use it?

Reply:
There will be a couple of follow-up PRs:

 - Allow updating core config while HA is running
 - Add a UI to update core config
 - Add core config setting to onboarding